advocacy_docs/supported-open-source/patroni/installing_on_debian.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Installing Patroni on Debian compatible systems'
+navTitle: 'Installing Patroni on Debian'
```

Comment:
This page isn't about installing Patroni, but is about installing the database server instead.